piton/design/chipset/rtl/packet_filter.v.pyv

```diff
@@ -317,28 +320,27 @@ for i in range(1, len(devices)):
     if info["name"] == "mem":
         outputstr = """
                 else begin
-`ifndef PITONSYS_NO_MC
-`ifdef PITON_FPGA_SYNTH
+//`ifndef PITONSYS_NO_MC
+//`ifdef PITON_FPGA_SYNTH
                     // route everything else to the memory when uart_boot_en is asserted
                     if ((noc2_filter_data[`MSG_ADDR_] >= `PHY_ADDR_WIDTH'h%x && noc2_filter_data[`MSG_ADDR_] < `PHY_ADDR_WIDTH'h%x + `PHY_ADDR_WIDTH'h%x) ||
                         (uart_boot_en)) begin
                         readdressed_flit0[`MSG_DST_X] = `NOC_X_WIDTH'h%x;
                     end else begin
-                        invalid_access_d = 1'b1;
+						            invalid_access = 1'b1;
                     end
-`else // PITON_FPGA_SYNTH
-                // route everything else to the memory in simulation
-                readdressed_flit0[`MSG_DST_X] = `NOC_X_WIDTH'h%x;
-`endif
-`endif
+//`else // PITON_FPGA_SYNTH
+//                // route everything else to the memory in simulation
+//                readdressed_flit0[`MSG_DST_X] = `NOC_X_WIDTH'h%x;
+//`endif
+//`endif
```

Comment:
We need to remember to remove these comments and replace them with a macro when we merge this PR. I'm a little worried we'll forget and then break sparc simulation